phpunit.xml.dist

```diff
@@ -24,5 +24,7 @@
 
         <server name="IS_DOCTRINE_ORM_SUPPORTED" value="true" />
         <server name="ESCAPE_JSON" value="true" />
+
+        <server name="KERNEL_CLASS" value="App\Kernel" />
```

Comment:
Is this change related to gulp? Looks like should have been done at 1.13 branch.

Reply:
`1.13` is working, idk why the build broke here. I'll check that again, I'll consider moving these two changes into a separate PR.

Reply:
I also checked this, and this change is a part of making the test app working [`ef923f9` (#15314)](https://github.com/Sylius/Sylius/pull/15314/commits/ef923f915162966dc0417d5dcfaa8e3da22e90fa). For some reason before it worked, and after adding the Webpack Encore it stopped, so it might also be considered as an integral part of this PR.

Reply:
I removed this line, while implementing Twig Components in Sylius I discovered the issue only exists when we try to test bundle's PHPUnit suite from the main vendor (not the package one) 😅.